[2025-04-25T05:10:44.940+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: user_processing.store_user manual__2025-04-25T05:10:07.541648+00:00 [queued]>
[2025-04-25T05:10:44.944+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: user_processing.store_user manual__2025-04-25T05:10:07.541648+00:00 [queued]>
[2025-04-25T05:10:44.944+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2025-04-25T05:10:44.944+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2025-04-25T05:10:44.944+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2025-04-25T05:10:44.949+0000] {taskinstance.py:1300} INFO - Executing <Task(PythonOperator): store_user> on 2025-04-25 05:10:07.541648+00:00
[2025-04-25T05:10:44.951+0000] {standard_task_runner.py:55} INFO - Started process 10456 to run task
[2025-04-25T05:10:44.953+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'manual__2025-04-25T05:10:07.541648+00:00', '--job-id', '22', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp86o8kihe']
[2025-04-25T05:10:44.953+0000] {standard_task_runner.py:83} INFO - Job 22: Subtask store_user
[2025-04-25T05:10:44.980+0000] {task_command.py:388} INFO - Running <TaskInstance: user_processing.store_user manual__2025-04-25T05:10:07.541648+00:00 [running]> on host b08efc87d46b
[2025-04-25T05:10:45.005+0000] {taskinstance.py:1509} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2025-04-25T05:10:07.541648+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2025-04-25T05:10:07.541648+00:00
[2025-04-25T05:10:45.007+0000] {postgres.py:158} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2025-04-25T05:10:45.010+0000] {base.py:73} INFO - Using connection ID 'postgres' for task execution.
[2025-04-25T05:10:45.012+0000] {python.py:177} INFO - Done. Returned value was: None
[2025-04-25T05:10:45.016+0000] {taskinstance.py:1323} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20250425T051007, start_date=20250425T051044, end_date=20250425T051045
[2025-04-25T05:10:45.052+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2025-04-25T05:10:45.060+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
